po/it.po

```diff
@@ -22,7 +22,7 @@ msgstr ""
 
 #: ../src/qml/NewGroupPage.qml:250
 msgid "#channelName"
```

Comment:
@lduboeuf can you please have a look at the source code? I doubt this word should be translated.

Reply:
@lduboeuf ping ^^

Reply:
i doubt it needs to be translated too,but in code you can see the translation function i18n.tr ... does it need to be fixed ?

Reply:
Edit: it is a placeholder text but i think it should be renamed "Channel Name" instead of #channelName